tests/phpunit/DefaultManifestGeneratorTest.php

```diff
@@ -162,7 +162,6 @@ public function testCsvWithoutHeaderCustomQuery(): void
             $builder = ColumnBuilder::create();
             $builder->setName($name);
             $builder->setType($type);
-            $builder->setPrimaryKey($name === 'pk1' || $name === 'pk2');
```

Comment:
Tady se to vlastně testovalo, jenomže to nedáválo smysl. V reálu ten PK nebude u custom query nasetovaný přes ColumnBuilder.